### ISSUE TITLE
x/build/cmd/gerritbot: Gobot shouldn't be added as a CC/Reviewer

### ISSUE THREAD

**Issue by mvdan** [golang/go#24382]
Gobot is added as a CC when it says "Trybots beginning", and as a Reviewer when it posts Trybot-Result+1 or Trybot-Result-1. For example, see: https://go-review.googlesource.com/c/go/+/100475

I realise that this is how Gerrit is supposed to work for humans - a user should be subscribed to updates if they comment, and added as a reviewer if they touch one of the flags/labels. However, I think this is noisy for bots. Is there a way to keep it from happening?

**Comment by andybons** [comment]
Are you referring to the fact that it’s even there in the list of CC/Reviewers as being noisy?

**Comment by mvdan** [comment]
Yes - and also how being added to those shows up in the CL comments.

**Comment by andybons** [comment]
This fix is needed but it will have to happen in Gerrit. This is just the way Gerrit currently works, with no configuration options (that I'm aware of) to suppress the behavior.

**Comment by andybons** [comment]
Thank you.
